### PR TITLE
Add Contact Form Section

### PR DESCRIPTION
**Pull request by luisrpavanelli**
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

We modified the html and css files, and also implemented formspree.

Index.html and syle.css:
- create a new brand with my collaborators
- create a contact form code in html and css according to figma
- correction of linters

formspree:
- create a formspree account
- create a code
- create and modify index.hmtl and style.css to place the formspree

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] reflects commit from only one partner in your pair-programming project. Every members of the team must at least have one commit in a group project. Kindly ensure every member contributes to the project with a commit message.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by luisrpavanelli** [comment]
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

File: style.css [hunk 1]
```diff
@@ -227,3 +227,28 @@ hr.roundeddivider {
...
+  /* Neutrals/N800 */
+
+  color: #172b4d;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```
Comment (shella12) [inline style.css:254]: - [ ] Please make sure that your `contact us` section looks like the provided [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=OrdN01rgF7wtPtkc-0) as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> You don't need to implement any functionality that requires JavaScript.
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Implement the button interactions (enable, hover, etc.).

_**Figma design**_ 

Screenshot 2023-02-22 at 3 26 58 AM

_**Your design**_

Screenshot 2023-02-22 at 3 18 16 AM

File: index.html [hunk 2]
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
+    <section>
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="">
+            <input type="text" name="user_name" placeholder="Full name" id="name" required>
```
Comment (shella12) [inline index.html:239]: - [ ] `Name` field input should not accept more than 30 characters as it is mentioned in  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) 

> Name field accepts up to 30 chars.

Kindly add  `maxlenght=30` attribute in the name text input field

File: index.html [hunk 3]
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
...
+            <input type="email" name="mail" placeholder="Email address" id="email" required>
+            <textarea name="message" id="msg" cols="30" rows="10" required placeholder="Enter text here"></textarea>
```
Comment (shella12) [inline index.html:241]: - [ ] Please make sure all input fields including the message field have `required` attribute set as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> A user cannot submit the form if one or more of the fields is empty.

- [ ] Kindly set `maxlength = 500` attribute for `textarea` message field as mentioned in the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> Message field accepts up to 500 chars.

File: index.html [hunk 4]
```diff
@@ -233,5 +233,14 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
+    <section>
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="">
```
Comment (shella12) [inline index.html:238]: - [ ] As stated in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements). you should use  `formspree` to collect data from your `contact` form 

> Collect data by using the [Formspree service](https://formspree.io/html).

Please make sure `Form` uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

- `action` value should be similar to `https://formspree.io/f/{form_id}`

- `method` value equals `POST`

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli and @smunoz1988 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- All inputs are inclosed in `form` ✅ 

## Required Changes ♻️

- [ ] _Check the comments under the review._


## Optional suggestions

- [ ] [OPTIONAL] Please add all contributors in to the author section of your README to give credit where credit is due


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
In this PR, we worked add contact form section, in html and css files, with collaborating @smunoz1988 and @Mylo16.

We modified the html and css files, and also implemented formspree.

Index.html and syle.css:
- create a new brand with my collaborators
- create a contact form code in html and css according to figma
- correction of linters


formspree:
- create a formspree account
- create a code
- create and modify index.hmtl and style.css to place the formspree

File: index.html [hunk 5]
```diff
@@ -233,5 +233,61 @@ <h3>Skills</h3>
...
+                <h2>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h2>
+            </div>
+        </div>
+    <form action="https://formspree.io/f/xlekbooe" method="post">
+        <div class="contactformcontainer">
+            <div class="contactcard">
+                <div class="text">
+                    <input type="text" name="user_name" placeholder="Full name" id="name" required>
```
Comment (souad988) [inline index.html:251]: [ optional ] kindly add max-length attribute with a value 30 as requested in this project requirements

File: index.html [hunk 6]
```diff
@@ -233,5 +233,61 @@ <h3>Skills</h3>
...
+                </div>
+                
+                <div class="email"></div>        
+                    <input type="email" name="mail" placeholder="Email address" id="email" required>
+                </div>
+                
+                <div class="message"></div>        
+                    <textarea name="message" id="msg" cols="30" rows="10" required placeholder="Enter text here"></textarea>
```
Comment (souad988) [inline index.html:259]: [ optional ] same here please don't forget to specify a text length limit as requested in this activity.

File: style.css [hunk 7]
```diff
@@ -227,3 +227,83 @@ hr.roundeddivider {
...
+
+.contactformcontainer:hover {
+  box-shadow: 0 5px 10px rgba(54, 179, 127, 0.8);
+}
+
+.contactformcontainer:active {
+  background-color: #008552;
+  box-shadow: none;
+}
```
Comment (souad988) [inline style.css:296]: I think you forgot to specify here that this properties should apply to the button not the container ,kindly check again and fix it also make sure to use the same shadow as in figma template  "0px 8px 16px rgba(54, 179, 127, 0.24)"

student shadow

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865

## Required Changes ♻️
_Check the comments under the review._
_ Also please don't forget to fix styling of your inputs as requested by the previous reviewer 

**your inputs:**
temp2 student contact form

**figma design**
figma temp2


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
Hello, I worked to make the modifies as you tell me to do, thank you so much!

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @luisrpavanelli ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- The form is working fine :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**